proto/dydxprotocol/perpetuals/perpetual.proto

```diff
@@ -96,7 +96,7 @@ message LiquidityTier {
   // The maximum position size at which the margin requirements are
   // not increased over the default values. Above this position size,
   // the margin requirements increase at a rate of sqrt(size).
-  uint64 base_position_notional = 5;
```

Comment:
Is there an upgrade handler needed for this change? Or will the protocol just ignore this value after the upgrade?

Reply:
No, since the field is only marked as deprecated so it's backwards compatible.